org.gnome.Totem.json

```diff
@@ -335,7 +335,6 @@
         {
             "name": "totem",
             "buildsystem": "meson",
-            "config-opts": ["-Denable-easy-codec-installation=no"],
```

Comment:
The build option isn't obsolete in the version of the software that's still in this file. The config option should be removed when the software is updated to the version that removes it. It's also still supported in the latest stable branch, so this commit should be dropped.